lib/std/net.zig

```diff
@@ -1191,7 +1191,7 @@ pub fn isValidHostName(hostname: []const u8) bool {
     if (hostname.len >= 254) return false;
     if (!std.unicode.utf8ValidateSlice(hostname)) return false;
     for (hostname) |byte| {
-        if (byte >= 0x80 or byte == '.' or byte == '-' or std.ascii.isAlNum(byte)) {
+        if (std.ascii.isASCII(byte) or byte == '.' or byte == '-' or std.ascii.isAlphanumeric(byte)) {
```

Comment:
I think it should be `!std.ascii.isASCII(byte)` here.
`std.ascii.isASCII(byte)` would be equivalent to `byte < 0x80`.

Reply:
Ah... I think you're right! Thanks for the find!